README.md

```diff
@@ -103,9 +103,13 @@ We have a [semantic versioning policy](docs/user-guide/semantic-versioning-polic
 
 [The MIT License](https://raw.githubusercontent.com/stylelint/stylelint/master/LICENSE).
 
+## Contributing Guidelines
+
+Check out the [Contributing Guidelines](CONTRIBUTING.md).
+
 ## Contributors
 
-This project exists thanks to [all these people](https://github.com/stylelint/stylelint/graphs/contributors). [Contribute](CONTRIBUTING.md).
+Without [all these wonderful people](https://github.com/stylelint/stylelint/graphs/contributors) this project won't cease to exist.
```

Comment:
```suggestion
Without [all these wonderful people](https://github.com/stylelint/stylelint/graphs/contributors) this project would cease to exist.
```

Reply:
That doesn't make any sense right :thinking:

Reply:
@ntwb that would mean just the opposite of what is intended :sweat_smile:

Reply:
Language is hard, here's my take, `won't` is the abbreviated form of `will not`, as such:
> _"Without all these wonderful people this project will not cease to exist."_

That's a double negate, `will not` and `cease` which is the actual opposite, it actually means that the project doesn't need the people to _exist_.

Whereas:

> _"Without all these wonderful people this project would cease to exist."_

The `would` in `would cease` states the outcome, what it will become, it `would no longer exist` , aka `would cease to exist`.

There's also lots of examples of this throughout the web:
> https://duckduckgo.com/?q="would+cease+to+exist"

There's also, though not so many examples
> https://duckduckgo.com/?q="won't+cease+to+exist"

Then, there's the rabbit hole here:
> https://english.stackexchange.com/search?q=would+cease+to+exist

Argh, again, langue is hard, English language even harder, and the English Stackexchange rabbit hole is deep, surprised I got out as quick as I did.

Reply:
I guess the sentence [here](https://github.com/madlabsinc/mevn-cli#why-should-i-contribute) within our project is also misleading right :thinking:

Reply:
Correct, that should also be updated in a follow up PR, or here, from:

> `Contributing helps people and makes the world simply a better place, without contributors this project would not cease to exist.`

To:

> `Contributing helps people and makes the world simply a better place, without contributors this project would cease to exist.`